docs/user/alerting/geo-alert-types.asciidoc

```diff
@@ -0,0 +1,131 @@
+[role="xpack"]
+[[geo-alert-types]]
+== Geo alert types
+
+experimental[] Two additional stack alerts are available for use by {kib} platinum license users:
```

Comment:
We shouldn't mention the license level in the docs.  Instead, the detail should go in https://www.elastic.co/subscriptions.  That is, get a row added in the alerting section of the tables there.

Then in the docs, you can either not mention that at all or else just say something like: "These stack alerts are available only in specific {subscriptions}." That attribute resolves to the link to the subscriptions page (so you have to get it added there first).

Reply:
I've removed this reference entirely for now, seems simplest for this PR. I'm happy to get that table updated though if you can point me in the right direction. Otherwise I can ask around about it!